_layouts/page.html

```diff
@@ -9,7 +9,7 @@
                     <div class="entry-header-wrap">  
                         {% if page.banner_image %}
                         <div class="post-thumbnail">
-                            <img src="{{ "/images/" | append: page.banner_image | prepend: site.baseurl }}" alt="{% if page.banner_image_alt %}{{ page.banner_image_alt }}{% else %}{{ page.title }}{% endif %}">
+                            <img class="post-thumbnail" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mNkYAAAAAYAAjCB0C8AAAAASUVORK5CYII=" style="background-image: url({{ "/images/" | append: page.banner_image | prepend: site.baseurl }});" alt="{% if page.banner_image_alt %}{{ page.banner_image_alt }}{% else %}{{ page.title }}{% endif %}">
```

Comment:
what is this src?

Reply:
1x1 transparent pixel
generated at http://png-pixel.com/